src/types/index.ts

```diff
@@ -6,7 +6,7 @@ import type { PageProps } from 'pages/types';
 import type React from 'react';
 import type { FunctionComponent, SVGProps } from 'react';
 
-export type NetworkName = 'polkadot' | 'kusama' | 'westend';
+export type NetworkName = 'creditcoin';
```

Comment:
I think if you just add `creditcoin` to the list of allowed values that will resolve the other typescript warnings in CI and possibly not have to delete certain code snippets for other networks. @nathanwhit WDYT ?

Reply:
Fixed